fullstory/src/main/java/com/tealium/remotecommands/fullstory/FullStoryInstance.kt

```diff
@@ -0,0 +1,21 @@
+package com.tealium.remotecommands.fullstory
+
+import com.fullstory.FS
+import com.fullstory.FSOnReadyListener
+import com.tealium.remotecommands.RemoteCommandContext
+
+class FullStoryInstance(private val remoteCommandContext: RemoteCommandContext) : FullStoryCommand {
+    override fun logEvent(eventName: String, eventData: Map<String, Any>?) {
+        FS.event(eventName, eventData)
+    }
+
+    override fun identifyUser(id: String, data: Map<String, Any>?) {
+        data?.let {
+            FS.identify(id, it)
```

Comment:
the `identify` method says it can accept nullable values for the `userVars` param, so would be easier to read this without the null checks:
`FS.identify(id, data)`